Skills.md

```diff
@@ -4,22 +4,22 @@ https://api.careerbuilder.com/core/tagging/skills
 
-| Parameter (4.1/4.2) | Required | Description |
+| Parameter (4.1/4.2/5.0) | Required | Description |
 |----------------|----------------|-----------------|
-| version        | Yes | Passed in via the Accept header. Possible values are "4.1" or "4.2". | 
+| version        | Yes | Passed in via the Accept header. Possible values are "4.1", "4.2", or "5.0". | 
 | content        | Yes | A string containing the content to be tagged |
 | language            | No | A string determining the language (total 22 languages supported) in which the input text is written. Default value is en. Note that the input parameter passing to language should be the language id. |
 | threshold      | No | A double value between 0 and 1 controlling minimum relevancy scores for skill recognition. Higher values will more tightly restrict the returned skill tags. Default is 0.5. A threshold of 0 means all seed skill phrases recognized by exact string matching will be returned. Note that this parameter is only supported for inputs in English. |
-| auto_thres     | No | A boolean value indicating whether automatic thresholding is desired. If enabled, then when input text contains 150 or fewer words, the threshold parameter will be overwritten to 0 and all confidence values will be overwritten to 1.0&#42;. Default is true.  Note that this parameter is only supported for inputs in en, fr, and de. |
+| auto_thres     | No | A boolean value indicating whether automatic thresholding is desired. If enabled, then when input text contains 150 or fewer words , the threshold parameter will be overwritten to 0 and all confidence values will be overwritten to 1.0&#42;. Default is true.  Note that this parameter is only supported for inputs in en, fr, and de. |
 | return_related_skills | No | A boolean value indicating whether to return related skills for each extracted skill. Defaults to false. Related skills contain the same fields as extracted skills, with confidence scores indicating their relevance to the extracted skill. |
 
 &#42; A more detailed explanation of this functionality: The tagger uses “context” to define semantic relevancy.  If the input is too short (<= 150 words) to constitute a “context,” the tagger by default returns everything by direct matching, resulting in 0.0 confidence scores due to lack of context for relatedness. To avoid confusion, a pseudo score “1” is assigned to indicate “directly matched.” This feature is called “auto thresholding” (auto_thres).
 
-The V4 taxonomy can be found through the TaxonomyAPI [here](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/TaxonomyService.md)
+The taxonomy can be found through the TaxonomyAPI [here](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/TaxonomyService.md)

```

Comment:
maybe "The taxonomies"?
I am not sure if v4.2 is added to TaxonomyService, probably not, I marked this as tech debt, just raise attention, do we consider to add v5 there after this is done?

Reply:
I imagine we should add v5 and v4.2, yes.

Reply:
I agree with Gavin's beef comment about pluralizing 'taxonomy'.